### DISCUSSION TITLE
Using public cloud (Vultr) block storage. Any advice?

### DISCUSSION THREAD

**Discussion by piersdd** [siderolabs/talos#8925] [Q&A] [answered]
I'm trying to provide workload storage to a test/dev cluster, running on Talos Linux 1.7.4 on Vultr.

The official [Storage | Talos Linux](https://www.talos.dev/v1.7/kubernetes-guides/configuration/storage/#public-cloud) documentation suggests 
> If you are running on a major public cloud, use their block storage. It is easy and automatic.

Can anyone point me in the direction of how to do such with Vultr block storage?  Or perhaps they're only referring to Amazon, Google & Microsoft.

Failing that, I've provisioned seperate block storage, attached to each node, perhaps to run OpenEBS/Mayastor or Longhorn.
Any recommendations for a super small cluster?
Thanks in advance, Piers

**Comment by Preisschild** [comment, marked as answer]
https://github.com/vultr/vultr-csi

You just can use the specific cloud providers Kubernetes CSI controller

**Reply by piersdd** [reply]
awesome. Thankyou Florian
